packages/engine/src/scene/components/ModelComponent.ts

```diff
@@ -81,6 +81,10 @@ export const ModelComponent = defineComponent({
   },
 
   onSet: (entity, component, json) => {
+    setComponent(entity, EnvmapComponent)
+    setComponent(entity, LoopAnimationComponent)
```

Comment:
we dont need to add envmap or loop animation components here, we dont actually want them by default